### PR TITLE
[Container API] Support getting single container from given account in ambry-frontend

### PR DESCRIPTION
**Pull request by jsjtzyy**
This PR makes changes in GetAccountsHandler to support getting single container if the
request path matches "/accounts/containers". This can avoid transmitting whole account
info when number of containers in this account is super large. The account id is also attached
to response header to help deserialize container.

### CODE REVIEW

**Review by SophieGuo410 — APPROVED** [review]
LGTM!

File: ambry-frontend/src/main/java/com/github/ambry/frontend/GetAccountsHandler.java [hunk 1]
```diff
@@ -152,5 +165,30 @@ private void start() {
       }
       return accounts;
     }
+
+    /**
+     * @return a pair of account and the container to get.
+     * @throws RestServiceException
+     */
+    private Pair<Account, Container> getContainer() throws RestServiceException {
```
Comment (lightningrob) [inline ambry-frontend/src/main/java/com/github/ambry/frontend/GetAccountsHandler.java:173]: Why does this method return account along with container?  I saw you are setting the account id in the response, but can't you get that from container.getParentAccountId()?

**Review by lightningrob — APPROVED** [review]
Approved with one comment.

File: ambry-frontend/src/main/java/com/github/ambry/frontend/GetAccountsHandler.java (hunk 1 above)
Comment (jsjtzyy) [reply ambry-frontend/src/main/java/com/github/ambry/frontend/GetAccountsHandler.java:173]: oh, that's true. Let me remove the account.

**Pull request merged**